tools/sv-report

```diff
@@ -557,13 +557,21 @@ try:
         report = jinja2.Template(
             f.read(), trim_blocks=True, lstrip_blocks=True)
 
+    try:
+        build_id = os.environ['GITHUB_ACTION']
+    except:
+        build_id = 'local'
+    datetime = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
I think it would be better to come up with a different variable name - right now this overwrites the module (the name of the veritable is the same as the module).

Maybe let's change it to `build_time` / `build_datetime`?